local/README.md

```diff
@@ -0,0 +1,37 @@
+# Locally run Anovos workloads without access to a Spark installation/cluster
+
+Note that this is currently an experimental feature and details are subject
+to change without prior notice.
+Once this way of running Anovos workloads is stable, we will publish the
+required Docker images on Docker Hub.
+For now, you'll need to build the image yourself.
+
+Build the `anovos-worker` image:
+
+```bash
+docker build . -t anovos-worker
+```
+
+You can specify the version of Apache Spark and Anovos using build args:
+```bash
+docker build --build-arg spark_version=3.2.1 --build-arg anovos_version=0.3.0 .
```

Comment:
Do we need to update this with every anovos version for `anovos_version`? Is there anyway for this to read the version from `version.py`, aka what you did above in the local workflow:
`export ANOVOS_VERSION=cat src/main/anovos/version.py | cut -d'"' -f 2`

Reply:
There is no need to update the README. It just demonstrates how one can build the image for different versions of Spark and Anovos, which most users will never do. They will just use the image we'll provide through Docker Hub. (See my comment regarding steps to take right before and right after the release of the next Anovos version.)

Reply:
Building the images for Docker Hub is handled in the GitHub Actions workflow

Reply:
I see. I think you can clarify it a bit more, something like: For example, for spark version 3.2.1 and anovos version 0.3.0, this is the command.

Reply:
It says "You can specify the version of Apache Spark and Anovos using build args" right above? I doubt that anyone would think that `anovos_version=0.3.0` means anything other than "use Anovos version 0.3.0", or am I missing some alternative interpretation here?

Reply:
Got it. Will mark this as resolved.